src/plugins/workspace/public/plugin.ts

```diff
@@ -109,19 +167,21 @@ export class WorkspacesPlugin implements Plugin<{}, {}, WorkspacesPluginSetupDep
   private workspaceToChromeNavLink(
     workspace: WorkspaceAttribute,
     workspacesStart: WorkspacesStart,
-    application: ApplicationStart
+    application: ApplicationStart,
+    index: number
   ): ChromeNavLink {
     const id = WORKSPACE_APP_ID + '/' + workspace.id;
     const url = workspacesStart?.formatUrlWithWorkspaceId(
       application.getUrlForApp(WORKSPACE_APP_ID, {
```

Comment:
```suggestion
      application.getUrlForApp(WORKSPACE_APP_ID + PATHS.overview, {
       absolute: true,
     }),
```

We should use the new applicationId in this case I think?